pkg/exporter/sumologicexporter/sender.go

```diff
@@ -98,19 +92,30 @@ func (b *bodyBuilder) Reset() {
 }
 
 // addLine adds line to builder and increments counter
-func (b *bodyBuilder) addLine(line string) error {
-	_, err := b.builder.WriteString(line)
-	if err != nil {
-		return err
-	}
+func (b *bodyBuilder) addLine(line string) {
+	b.builder.WriteString(line) // WriteString can't actually return an error
 	b.counter += 1
-	return nil
+}
+
+// addLine adds multiple lines to builder and increments counter
+func (b *bodyBuilder) addLines(lines []string) {
+	if len(lines) == 0 {
+		return
+	}
+
+	// add the first line separately to avoid a conditional in the loop
+	b.builder.WriteString(lines[0])
+
+	for _, line := range lines[1:] {
```

Comment:
just checking, does it work as expected for one item list?

Reply:
Yeah, it's an empty slice then and we don't do anything.